lib/shared/navigation_helper.dart

```diff
@@ -1,5 +1,9 @@
 extension LinkWithTabInfo on String {
-  String withTabArg(int tabIndex) {
+  String withTabArg(int? tabIndex) {
```

Comment:
This makes sense and works. For some reason it feels like a use case that should be covered by auto_router, no?

Maybe a combo of using AutoRouter / AutoTabsRouter and `.back()` history. Not sure.